app/src/lib/git/push.ts

```diff
@@ -67,16 +69,15 @@ export async function push(
     remoteBranch ? `${localBranch}:${remoteBranch}` : localBranch,
   ]
 
+  if (tagsToPush !== null) {
+    args.push(...tagsToPush)
+  }
   if (!remoteBranch) {
     args.push('--set-upstream')
   } else if (options.forceWithLease === true) {
     args.push('--force-with-lease')
   }
 
-  if (enableGitTagsCreation()) {
-    args.push('--follow-tags')
-  }
```

Comment:
I removed the check for the feature flag, since if the feature flag is disabled the array of tags will get null